projects/project-player-library/src/lib/assets/i18n/en.json

```diff
@@ -106,5 +106,10 @@
     "FILE_UPLOAD_ERROR": "Error in uploading files",
     "FILE_UPLOAD_ERROR_MSG": "Some files seem to be corrupted or deleted. Re-upload files to proceed.",
     "VIEW": "View",
-    "FILE_CORRUPTED": "File corrupted"
+    "FILE_CORRUPTED": "File corrupted",
+    "CERTIFICATE_LABLE":"Certificate",
+    "PDF":"pdf",
+    "PNG":"png",
+    "CERTIFICATE_CRETIREIA_BFR_SUBMITTING":"Please ensure you have met the certificate criteria for this improvement project before submitting.",
```

Comment:
Check the spelling and change "BFR" to the whole word

Reply:
done